ironfish-cli/src/commands/ceremony.ts

```diff
@@ -46,11 +47,21 @@ export default class Ceremony extends IronfishCommand {
 
     // Prompt for randomness
     let randomness: string | null = await CliUx.ux.prompt(
-      `If you'd like to contribute your own randomness to the ceremony, type it here, then press Enter. For more information on where this should come from and its importance, please read https://setup.ironfish.network. If you'd like the command to generate some randomness for you, just press Enter:`,
+      `If you'd like to contribute your own randomness to the ceremony, type it here, then press Enter. For more information on where this should come from and its importance, please read https://setup.ironfish.network. If you'd like the command to generate some randomness for you, just press Enter`,
       { required: false },
     )
     randomness = randomness.length ? randomness : null
 
+    let name = 'anonymous'
+    do {
+      // Prompt for graffiti / name
+      const nameInput = await CliUx.ux.prompt(
+        `\nIf you'd like to associate a name with this contribution, type it here, then press Enter. Name must only include letters, numbers and spaces and be less than 30 characters. To accept the default, just press Enter (anonymous)`,
```

Comment:
nit:  it might be better to phrase like `To accept the default as anonymous, just press Enter`

Reply:
Maybe "To contribute anonymously, just press Enter" or "Otherwise, to contribute anonymously, just press Enter". I do agree it's a little bit confusing what the default is

Reply:
Yea good suggestions